integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/model/BridgeUsingPropertyMarkerAccessIT.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.integrationtest.mapper.orm.model;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.search.engine.backend.document.model.dsl.Store;
+import org.hibernate.search.engine.spatial.GeoPoint;
+import org.hibernate.search.engine.spatial.ImmutableGeoPoint;
+import org.hibernate.search.mapper.pojo.bridge.builtin.spatial.annotation.GeoPointBridge;
+import org.hibernate.search.mapper.pojo.bridge.builtin.spatial.annotation.Latitude;
+import org.hibernate.search.mapper.pojo.bridge.builtin.spatial.annotation.Longitude;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.DocumentId;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.Indexed;
+import org.hibernate.search.util.impl.integrationtest.common.rule.BackendMock;
+import org.hibernate.search.util.impl.integrationtest.orm.OrmSetupHelper;
+import org.hibernate.search.util.impl.integrationtest.orm.OrmUtils;
+import org.hibernate.search.util.impl.test.annotation.TestForIssue;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+/**
+ * Test that a bridge using markers to retrieve and access properties
+ * will behave as expected regarding access mode.
+ *
```

Comment:
A `<p>` would be in order (even if not an API, I think it's better to have a proper Javadoc).

Reply:
Right, done.